css-variables-and-js/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet">
+    <title>Photo editor online.</title>
+</head>
+<body>
+    <header>
+        <h1 class="visually-hidden">Photo editor.</h1>
+    </header>
+    <main>
+        <div class="photo-editor">
+            <div class="photo-area">
+                <div class="photo-area__photo">
+                    <img class="photo-area__image" src="./assets/images/photo-editor.jpg" alt="dog">
+                </div>
+            </div>
+            <div class="editor-area">
+                <!-- <div class="editor-area__wrapper"> -->
+                    <h2 class="editor-area__title title">Editor</h2>
+                    <hr>
+                    <div class="editor-area__filter filter">
+                        <button class="filter__button button" data-change-type="background">
+                            <img class="button__icon" src="./assets/icons/icon-background.jpg" height="50px" width="50px" alt="icon-background">
+                            <div class="button__discription">background</div>
```

Comment:
use `span` instead of `div`, because content is simple text. And pls fix it in whole html file